### ISSUE TITLE
stack_metadata file is redundant

### ISSUE THREAD

**Issue by atpollmann** [atpollmann/cf-cli-helper#20]
In the origins, this file was meant for information about the stack, but its contents are needed within the templates itself and therefore are duplicated in most projects.

The configuration can be simplified. All four necessary keys
```yaml
  "name": "veti-infrastructure",
  "domain": "veti",
  "version": "1.0",
  "bucket": "kms-cf-templates"
```
can be reside and be obtained from the ```stack_parameters``` file

**Comment by atpollmann** [comment]
Fixed in 4e25505662943cd386ad433ef3ba787c0665bfb7

**Comment by atpollmann** [comment]
It's missing the call to check metadata from the command itself
